File: lib/antlr-2.7.5/examples/cpp/parseBinary/Main.cpp

#include <iostream>

#include "DataLexer.hpp"
#include "DataParser.hpp"

int main(int argc,char* argv[])
{
	ANTLR_USING_NAMESPACE(std)
	ANTLR_USING_NAMESPACE(antlr)
	try {
		DataLexer lexer(cin);
		DataParser parser(lexer);
		parser.file();
	}
	catch( ANTLRException& e )
	{
		cerr << "exception: " << e.getMessage() << endl;
		return -1;
	}
	catch( exception& e )
	{
		cerr << "exception: " << e.what() << endl;
		return -1;
	}
	return 0;
}

File: lib/antlr-2.7.5/examples/java/parseBinary/Main.java
import java.io.*;

class Main {
	public static void main(String[] args) {
		try {
			DataLexer lexer = new DataLexer(new DataInputStream(System.in));
			DataParser parser = new DataParser(lexer);
			parser.file();
		} catch(Exception e) {
			System.err.println("exception: "+e);
		}
	}
}

